programs/openbook_v2/tests/program.rs

```diff
@@ -0,0 +1,32 @@
+use std::fs;
+use solana_program::pubkey::Pubkey;
+use solana_program::rent::Rent;
+use solana_program_test::ProgramTest;
+use solana_sdk::account::Account;
+use crate::utils::get_paths;
+
+pub fn setup_programs(validator: &mut ProgramTest) -> anyhow::Result<()>{
+    let path = get_paths();
+    // add openbook v2
+    add_program(validator, openbook_v2_light::id(), &format!("{}/deps/openbook_v2.so", path))?;
+    // add drift
+    add_program(validator, drift::id(), &format!("{}/target/deploy/drift.so", path))?;
+    // add pyth
+    add_program(validator, pyth::id(), &format!("{}/target/deploy/pyth.so", path))?;
+    Ok(())
+}
+
+pub fn add_program(validator: &mut ProgramTest, id: Pubkey, path: &str ) -> anyhow::Result<()> {
```

Comment:
i'd just `solana program dump <OPENBOOK_ID>` into a fixtures directory & use that & just use target/deploy as the OUT_DIR and source the rest of the .so files from there

Reply:
yeah, i will change it, but before few commits ago you had .so files in the deps directory,

Reply:
because for me - there was not clear - how to setup test in the manner - that openbook_v2.so file is in the different directory as drift.so, also i was not quite sure how to setup in way - that it will run on my PCs and in your github ci/cd